network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/article_hero.html

```diff
@@ -0,0 +1,73 @@
+{% load static i18n wagtailimages_tags homepage_tags%}
+
+{% image page.hero_image fill-1400x675 as bg_img %}
+
+{% with control_btn_classes="ml-auto tw-opacity-80 hover:tw-opacity-100 tw-hidden" full_screen_content_classes="tw-absolute tw-min-w-full tw-min-h-full tw-max-w-none tw-top-0" two_col_content_classes="tw-h-full tw-w-full tw-object-center tw-object-cover" %}
+
+
+{% if page.hero_layout != "full_screen" %}
```

Comment:
This template handles all 3 of the updated article hero layouts (full screen/Image left/image right)

If the layout choice is either "image left" or "image right" or more simply "not full screen," then render a two column responsive layout.


The "else" block below renders the 1 column, full screen image/video layout.